scripts/mender-qemu

```diff
@@ -31,18 +31,21 @@ else
     IMAGE_NAME=core-image-full-cmdline
 fi
 
-if [ -d ${BUILDDIR}/tmp-glibc ]; then
+if [ -d "${BUILDDIR}/tmp-glibc" ]; then
     BUILDTMP=${BUILDDIR}/tmp-glibc
 else
     BUILDTMP=${BUILDDIR}/tmp
 fi
 
+UBOOT_ELF=${UBOOT_ELF:="${BUILDTMP}/deploy/images/vexpress-qemu/u-boot.elf"}
+VEXPRESS_SDIMG=${VEXPRESS_SDIMG:="${BUILDTMP}/deploy/images/vexpress-qemu/${IMAGE_NAME}-vexpress-qemu.sdimg"}
+
 QEMU_AUDIO_DRV=none \
-    ${BUILDTMP}/sysroots/x86_64-linux/usr/bin/qemu-system-arm \
+    qemu-system-arm \
```

Comment:
I changed this in a later commit:

https://github.com/GregorioDiStefano/meta-mender/blob/509761d1eab721fd3ca6f3ad9aefdacac2a12362/scripts/mender-qemu